zrp/data/processed/standard_population_weights.json

```diff
@@ -0,0 +1 @@
+[{"WHITE_ALONE": 0.704171541435853, "BLACK_OR_AFRICAN_AMERICAN_ALONE": 0.126243902871607, "AMERICAN_INDIAN_AND_ALASKA_NATIVE_ALONE": 0.00823290472845048, "ASIAN_ALONE": 0.0564095784530982, "NATIVE_HAWAIIAN_AND_OTHER_PACIFIC_ISLANDER_ALONE": 0.00187220288319317, "SOME_OTHER_RACE_ALONE": 0.0513946460639222, "TWO_OR_MORE_RACES": 0.0516752235638751}, {"WHITE_ALONE": 0.785090741843671, "BLACK_OR_AFRICAN_AMERICAN_ALONE": 0.140751100444378, "AMERICAN_INDIAN_AND_ALASKA_NATIVE_ALONE": 0.00917898111532281, "ASIAN_ALONE": 0.0628918313065141, "NATIVE_HAWAIIAN_AND_OTHER_PACIFIC_ISLANDER_ALONE": 0.00208734529011336}, {"WHITE": 0.522399958663599, "HISPANIC": 0.181771582772255, "BLACK": 0.126243902871607, "AIAN": 0.00823290472845048, "ASIAN": 0.0564095784530982, "PI": 0.00187220288319317, "OTHER": 0.103069869627797}, {"WHITE": 0.582431051175431, "HISPANIC": 0.20265969066824, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "ASIAN": 0.0628918313065141, "PI": 0.00208734529011336}, {"WHITE": 0.522399958663599, "HISPANIC": 0.181771582772255, "BLACK": 0.126243902871607, "AIAN": 0.00823290472845048, "AAPI": 0.0582817813362914, "OTHER": 0.103069869627797}, {"WHITE": 0.582431051175431, "HISPANIC": 0.20265969066824, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "AAPI": 0.0649791765966274}, {"WHITE": 0.704171541435853, "BLACK": 0.126243902871607, "AIAN": 0.00823290472845048, "ASIAN": 0.0564095784530982, "PI": 0.00187220288319317, "OTHER": 0.103069869627797}, {"WHITE": 0.785090741843671, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "ASIAN": 0.0628918313065141, "PI": 0.00208734529011336}, {"WHITE": 0.704171541435853, "BLACK": 0.126243902871607, "AIAN": 0.00823290472845048, "AAPI": 0.0582817813362914, "OTHER": 0.103069869627797}, {"WHITE": 0.785090741843671, "BLACK": 0.140751100444378, "AIAN": 0.00917898111532281, "AAPI": 0.0649791765966274}]
```

Comment:
Looks like we are missing the two or more races category most of the time. Also looks like we either need a mapping of long names to short names or additional records to this list.

Please add

-  abbreviations for the first item of the list ['WHITE', 'BLACK', 'AIAN', 'ASIAN', 'PI', 'OTHER', 'MULTI']
- additional lists where two or more races & other are separated

Please define the abbreviation for two or more races is it 'multi', 'tom' '2plus' 2ormore', etc

Reply:
New version includes all of the corrections mentioned above. I decided to keep short versions only - the long ones seem to add extra level of complexity for no reason imo.